nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -119,6 +119,11 @@ package object runtime {
    */
   @noinline def loop(): Unit = ExecutionContext.loop()
 
+  /** Run the runtime's event loop once and return true when internal queue has
+   *  more Runnable to run.
+   */
+  @noinline def loopRunOnce(): Boolean = ExecutionContext.loopRunOnce()
```

Comment:
Can we use some more meaningful name? Maybe some out of 
`dequeueAndExecute`, `runNext`, `execute`, `interleave` or maybe something else. `loopRunOnce` doesn't seem to be a good name. 
Also would be useful to also port the tests from the original PR. Especially to ensure we don't use multithreaded execution context by mistake.